src/utils/vaultMath.ts

```diff
@@ -49,7 +49,13 @@ export const computeToLock = (
   toReceive: NatValue,
   defaultCollateralization: Ratio,
   loanFee: Ratio,
+  // Round up by default to preserve offer safety. But, if using this to set
+  // max input, round down to avoid hitting debt limit.
+  roundDown: boolean = false,
```

Comment:
technically it's not rounding. Let's be more explicit about ceil vs floor. https://github.com/Agoric/agoric-sdk/blob/bcf33feb2ff38d08714c77ef8e644409a4d894fa/packages/zoe/src/contractSupport/safeMath.js#L21-L27

Floor is the default way JS does bigint division (truncating non-integer) so it should be the default. 

```suggestion
  remainderHandling: 'floor' | 'ceil' = 'floor',
```

Reply:
Done